geoips/cli.py

```diff
@@ -112,15 +112,12 @@ def get_interface(name):
         return getattr(interfaces, name)
     except AttributeError:
         try:
-            return getattr(stable, name)
+            return getattr(dev, name)
         except AttributeError:
-            try:
-                return getattr(dev, name)
-            except AttributeError:
-                raise AttributeError(
-                    f'Interface "{name}" not found in '
-                    "either stable or developmental interface sets"
-                )
+            raise AttributeError(
+                f'Interface "{name}" not found in '
+                "either stable or developmental interface sets"
```

Comment:
Should comment be updated for clarity since no Stable option exists anymore?

Reply:
@christopher-selman Glad to see you're paying attention!